provisioning_client/tools/symm_key_provision/symm_key_provision.c

```diff
@@ -20,7 +20,14 @@ typedef struct REGISTRATION_INFO_TAG
 
 static char* get_user_input(const char* text_value, int max_len)
 {
-    char* result = malloc(max_len + 1);
+    int result_len = max_len + 1;
+    if (result_len <= 1)
+    {
+        (void)printf("invalid max_len size\r\n");
+        return NULL;
```

Comment:
It breaks our style of single point of return.

Reply:
line 34 already does an early return